main.cc

```diff
@@ -1089,6 +1092,17 @@ int main(int ac, char** av) {
                 startlog.info("Alternator server listening on {} port {}", addr, cfg->alternator_port());
             }
 
+            if (cfg->enable_redis_protocol()) {
+                supervisor::notify("starting create redis databases if not exists.");
+                redis::create_keyspace_if_not_exists(cfg).get();
```

Comment:
alternator has a similar method `executor::maybe_create_keyspace()` in alternator/executor.cc

And that's included in start() stage.

Reply:
Thanks. I will read the code of executor::maybe_create_keyspace. Maybe it 's useful for me.

Reply:
Thanks for your suggestion.

Reply:
It's different from the case of Alternator. It's better to create the keyspaces/tables in the start stage before the server serving the requests.